### PR TITLE
fix incorrect metrics output by metrics-curl.rb

### PR DESCRIPTION
**Pull request by zanoubia**
## Pull Request Checklist

**Is this in reference to an existing issue?**
#54
#### General
- [ ] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)
- [ ] Update README with any necessary configuration snippets
- [ ] Binstubs are created if needed
- [ ] RuboCop passes
- [ ] Existing tests pass 
#### New Plugins
- [ ] Tests
- [ ] Add the plugin to the README
- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

**Comment by eheydrick** [comment]
Hi thanks for the PR, can you resolve the rubocop issues and we'll get this merged. https://travis-ci.org/sensu-plugins/sensu-plugins-http/jobs/165249886. Can you also add an entry to the CHANGELOG under Unreleased.

File: lib/sensu-plugins-http/version.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 module SensuPluginsHttp
   module Version
-    MAJOR = 1
+    MAJOR = 2
```
Comment (GhostLyrics) [inline lib/sensu-plugins-http/version.rb]: Please do not change the version yourself. The sensu-plugins team does that when releasing a new version of the plugin.

**Comment by majormoses** [comment]
@zanoubia are you planning on getting this over the line? If I don't hear back from you in the next week or so I will close this out.

**Comment by majormoses** [comment]
closing due to inactivity, please feel free to re-open if you plan on getting this over the line.

**Pull request closed without merging**